src/examples/table/TableDraggable.tsx

```diff
@@ -5,109 +5,37 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { PropsWithChildren, useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import styled from 'styled-components';
-import { DragDropContext, Droppable, Draggable, DropResult } from 'react-beautiful-dnd';
-import { Table, Row, Cell, Head, HeaderCell, HeaderRow, Body } from '@zendeskgarden/react-tables';
+import { Table } from '@zendeskgarden/react-tables';
 import { ReactComponent as GripIcon } from '@zendeskgarden/svg-icons/src/12/grip.svg';
-import { getColorV8 } from '@zendeskgarden/react-theming';
-
-const DraggableRow = styled(Row)<{ isDraggingOver: boolean }>`
-  ${props =>
-    props.isDraggingOver
-      ? `
-  :hover {
-    background-color: inherit !important;
-  }
-`
-      : ''}
-`;
+import { getColor } from '@zendeskgarden/react-theming';
+import {
+  DndContext,
+  closestCenter,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors,
+  DragEndEvent
+} from '@dnd-kit/core';
+import {
+  arrayMove,
+  SortableContext,
+  sortableKeyboardCoordinates,
+  useSortable,
+  verticalListSortingStrategy
+} from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
 
 const DraggableContainer = styled.div`
-  color: ${props => getColorV8('primaryHue', 600, props.theme)};
-
-  &:hover {
-    color: ${props => getColorV8('primaryHue', 700, props.theme)};
-  }
+  cursor: grab;
```

Comment:
Please update the cursor to `grabbing` on drag-drop. See https://garden.zendesk.com/components/table#draggable for details

Reply:
It also looks like there's missing selected (light blue background) styling for the keyboard `<space>` action.

Reply:
Fixed in [7f8796c](https://github.com/zendeskgarden/website/pull/624/commits/7f8796cd17337d952310a0c47efa9381c78801fd) ✅ 

![Screenshot 2024-08-15 at 12 11 53 PM (2)](https://github.com/user-attachments/assets/a9e48656-5666-4ea8-925a-9ae2aca1a195)